wagn/lib/wagn/parser.rb

```diff
@@ -0,0 +1,93 @@
+# -*- encoding : utf-8 -*-
+
+module Wagn
+  class Parser
+    class << self
+      def db_task command, envs
```

Comment:
Unused method argument - `envs`. If it's necessary, use `_` or `_envs` as an argument name to indicate that it won't be used.<br>Method has too many lines. [21/15]